op-scim-bridge/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Should this file be at the root of the repo? Or in `op-scim-bridge`?

Reply:
Thanks for checking. The current location is intentional. I also double checked with `1Password/connect-helm-charts` to be sure.